src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -89,6 +90,7 @@ public async Task<ActionResult> GetInstanceOwners(int instanceOwnerPartyId)
         /// <param name="size">the page size</param>
         /// <returns>list of all instances for given instance owner</returns>
         /// <!-- GET /instances?org=tdd or GET /instances?appId=tdd/app2 -->
+        [Authorize(Policy = "InstanceRead")]
```

Comment:
I dont think we can use standard if this returns multiple

Reply:
I agree, removing the tag.

Reply:
@Kritsi is implementing a custome authorization for this endpoint